### ISSUE TITLE
Make the checks that query the casper client run only once

### ISSUE THREAD

**Issue by cspramit** [casper-network/casper-node#2723]
Make the checks that query the casper client run only once (this requires prior merge of the convenience methods from[ here](https://github.com/casper-network/casper-test/pull/65)

**Comment by devendran-m** [comment]
We need to figure out what happened in the below ticket and PR to understand whether the changes are required.
- casper-test [PR#65](https://github.com/casper-network/casper-test/pull/65)
- casper-tect [Issue#14](https://github.com/casper-network/casper-test/issues/14)

**Comment by devendran-m** [comment]
This refers to a test repo that we no longer support, so it's a won't do.
